docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -270,19 +13,42 @@ This section describes the functionality of the Dasharo Tools Suite. These are:
     + [run commands from iPXE shell automatically](#run-commands-from-ipxe-shell-automatically),
     + [run DTS using VentoyOS](#run-dts-using-ventoyos).
 
-### Dasharo zero-touch initial deployment
+## Available Commands
+
+When DTS is started, it has following options for the user to choose from:
+
+* **(1)** Dasharo HCL Report:
+
+  Generate HCL report on the current machine.
```

Comment:
You could just use:
* **1)** [Dasharo HCL Report](#hcl-report) - generate Hardware Compatibility List Report

Reply:
Covered by: b133540c400db01df4c8bcaf581f2103c23cd870